library/blivet.py

```diff
@@ -152,12 +152,17 @@ class BlivetAnsibleError(Exception):
 class BlivetBase(object):
     blivet_device_class = None
     _type = None
+    _volume = None
```

Comment:
Use `self._spec_dict` in the base class. You only have `self._volume` in the volume subclasses and `self._pool` in the pool subclasses.

Reply:
Agree. The reason of `_volume` being used here is that some `BlivetBase` methods work with `self._volume` and pylint doesn't like it since it is undefined. Other way to fix this would be to disable pylint warnings. I think that using 'abstract variable' is slightly better solution because it handles all the future `self._volume` invocations from this class.
I should probably add some explanation into the code, though.

Reply:
I have eventually decided to disable pylint. Since dummy variable hadn't had all required attributes of the `self._volume`, pylint noticed it anyways. This also nullifies my previous argument about future invocations.